cwf/cloud/go/services/cwf/analytics/analytics.go

```diff
@@ -69,58 +68,59 @@ func GetAnalyticsCalculations(config *cwf_service.Config) []calculations.Calcula
 
 	allCalculations := make([]calculations.Calculation, 0)
```

Comment:
I know this was from preexisting code: golang style prefers brevity for much-used variables with clear purpose. So e.g. consider renaming `allCalculations` to `calcs` for increased readability of the below blocks